app/org/sagebionetworks/bridge/services/AuthenticationService.java

```diff
@@ -116,13 +127,21 @@ public void requestEmailSignIn(SignIn signIn) {
             throw new LimitExceededException("Email currently pending confirmation.");
```

Comment:
I need to remove this. If the email fails to send and the user tries to trigger another email, it just seems unreasonable to throw an error rather than resending the email. This isn't a good way to rate-limit requests.